torchaudio/datasets/cmuarctic.py

```diff
@@ -148,7 +148,7 @@ def __init__(self,
             if not os.path.isdir(self._path):
                 if not os.path.isfile(archive):
                     checksum = _CHECKSUMS.get(url, None)
-                    download_url(url, root, hash_value=checksum, hash_type="md5")
+                    download_url_to_file(url, root, hash_prefix=checksum)
```

Comment:
`download_url_to_file` can only handle SHA256 hash, therefore the MD5 sum cannot be passed to the function. 
Either the checksum needs to be updated to SHA256 or validation has to be performed separately after the download.